projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -6481,16 +6480,14 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
             const overlaySettings: OverlaySettings = {
```

Comment:
Loading overlaySettings could be moved to a private member like the `rowEditSettings` instead of creating them on each `open` call.